.travis.yml

```diff
@@ -1,3 +1,16 @@
-script:
-- docker build --target all_tests -t test . && docker run --name todo-app-test --env TRELLO_URL --env TRELLO_KEY --env TRELLO_BOARD_ID --env TRELLO_TOKEN test
-- docker build --target prod -t prod .
+jobs:
+  include:
+    - stage: test
+      script:
+        - docker build --target all_tests -t test .
+        - docker run --name todo-app-test --env TRELLO_URL --env TRELLO_KEY --env TRELLO_BOARD_ID --env TRELLO_TOKEN test
```

Comment:
This is actually relevant to module 7 rather than 8, but anyway:

By running all of your tests in a single command with the environment variables set like this means you are relying on the order of the tests. If the integration tests ran before the e2e tests, they would override the env vars and the e2e test would not be able to reset them within the `poetry run pytest` process.